kong/clustering/control_plane.lua

```diff
@@ -58,6 +58,21 @@ local REMOVED_FIELDS = require("kong.clustering.compat.removed_fields")
 local _log_prefix = "[clustering] "
 
 
+local function handle_export_deflated_reconfigure_payload(self)
+  local ok, p_err, err = pcall(self.export_deflated_reconfigure_payload, self)
+  if not ok then
+    -- hard error -> ok == false, p_err has an error message
+    return false, p_err
+  end
+  if not p_err then
+    -- fail from typical return nil, err -> ok == true, p_err == nil, err has an error message
+    return false, err
+  end
+  -- succeeded -> ok == true, p_err == nil, err == nil
+  return true
```

Comment:
I think this section can be simplified to 
```suggestion
  return ok, p_err or err
```

Reply:
https://github.com/Kong/kong/pull/8668/commits/ebf080739312a08756664079ad05d03eb40cd1f9